README.md

```diff
@@ -84,10 +84,10 @@ However, `kcp` is not an acronym.
 
```

Comment:
While you're in here, would you mind fixing the typo on [L68](https://github.com/kcp-dev/kcp/pull/52/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R68)? s/reconiling/reconciling/

Reply:
good spotting, fixed now